### PR TITLE
hide-module error fix in step 4

### PR DESCRIPTION
**Pull request by Ranjeet-Kalsi**
Targets #3088 

Some changes in installation guide  step 4 
error fixed    --hide-module

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -44,6 +44,15 @@ Before you start following the guidelines, make sure to go through the [prerequi
    npm install
    npm run dev
    ```
+    If you are still getting the error --hide-module then revert all changes by git checkout.
```
Comment (GauravGusain98) [inline docs/installation.md:47]:
```suggestion
    If you are still getting the error --hide-module then revert all changes by `git checkout .`
```

File: docs/installation.md [hunk 2]
```diff
@@ -44,6 +44,15 @@ Before you start following the guidelines, make sure to go through the [prerequi
...
+
+	**Note:** Check the version of laravel-mix in package-lock.json it should be 5.0.9
+    ```sh
+	npm install
+    npm run dev
```
Comment (GauravGusain98) [inline docs/installation.md:52]:
```suggestion
        npm run dev
```

File: docs/installation.md [hunk 3]
```diff
@@ -44,6 +44,15 @@ Before you start following the guidelines, make sure to go through the [prerequi
...
+    ```
+    **Note:** if laravel-mix version in package-lock.json changes to version 6 then npm run dev will throw error,
+    so it should be at version 5 only.
```
Comment (GauravGusain98) [inline docs/installation.md:55]:
```suggestion
    **Note:** If laravel-mix version in package-lock.json changes to version 6 then npm run dev will throw error, so it should be at version 5 only.
```

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]

File: docs/installation.md (hunk 1 above)
Comment (Ranjeet-Kalsi) [reply docs/installation.md]: ok sir i will change it

File: docs/installation.md (hunk 3 above)
Comment (Ranjeet-Kalsi) [reply docs/installation.md]: Sir changes applied, please review

Comment (Ranjeet-Kalsi) [reply docs/installation.md]: hide module
Changes applied 
guide preview attached

**Review by GauravGusain98 — APPROVED** [review]
LGTM

**Pull request merged**